### ISSUE TITLE
Unable to delete playing file from volumes w/o trash

### ISSUE THREAD

**Issue by x43x61x69** [iina/iina#2571]
**System and IINA version:**

- macOS 10.15 (19A536g)
- IINA 1.0.4

**Expected behavior:**

File got deleted.

**Actual behavior:**

IINA would fail to delete the file if the volume has no trash enabled (usually mounted remote drive):
<img width="532" alt="img" src="https://user-images.githubusercontent.com/1477686/63493745-cea9b600-c4ee-11e9-9f46-4a20c9ed11a5.png">

**Comment by anohren** [comment]
It's not apparent to me what the net benefit would be of allowing IINA to perform an irreversible destructive operation like that. AFAIK there's not even a confirmation dialog before a delete which would make it worse.

Better would be to gray out the menu entry of there's no trash. Allowing file deletions anyway would at the very least require an advanced setting to opt in to dangerous behaviour as well as a confirmation dialog before every deletion.

**Comment by x43x61x69** [comment]
It refused to delete because IINA believes the volume has no trash, but it could have just not the macOS way. For example, if you have a Synology NAS and store all your videos remotely to they can be shared with all devices, you won't be able to delete them simply because Synology has it's own version of trash/recycle bin system.

And I do think it's better to have a confirmation before the actual deletion or can only be opt-in via Advanced settings. Just that unable to delete the file because you can't "trash" it the macOS way makes no sense in 2019.

**Comment by anohren** [comment]
I think the only reason the feature exists at all is because the author found that it could be done safely by using Apples libraries to do the actual move to trash in a macOS environment. While it would be nice to have IINA be aware of other recycle bin systems it would probably be to much effort to implement safely, and as for deleting it outright, sounds like a good idea to implement after perhaps moving all of this file management business into a future plugin. 

On a side note, personally I think that for an app whose responsibilities includes neither creating, managing, organising or moving media files it is even out of place for it to move files to the trash — no matter which year it is :) It's already to easy to move to the trash unintentionally as it is, considering the placement of the delete option in the file menu. I've even done tests to make sure it doesn't delete anything on my SMB shares, since I'd never forgive an app that did that to me just because my accuracy was off by 10 pixels. 

Tldr: let's tread carefully here considering the damage that can be made is so much greater than the convenience gained.

**Comment by WillemWever** [comment]
You can use a keyboard shortcut to solve this: @iina delete-current-file-hard

Unfortunately the keyboard shortcut stoped working at my Mac :<

**Comment by x43x61x69** [comment]
@WillemWever I use `@iina delete-current-file` and it worked! Thanks a lot!
